### ISSUE TITLE
Test Projects dont work

### ISSUE THREAD

**Issue by bettina4you** [achlubek/vengine#1]
dear achlubek,

i know that the vengine C# Version is no more under development, but i would very, very much like to get it running, because i searched for a long time for a c# game engine like yours.

I have one little experience with c++

My Settings: Win7 SP1 64bit  VS2015

The code compiles good, but after trying the demo scene files none of them is running.

The import of the scene files have problems.

e.g. the scene file has a dicard_texture but the reader does not know this kind of texture
      or the material definitions color has 4 attributes but the reader wants them to be exactle only 3 (r,g,b)
      other demo refers to a sponza.obj file but this is not included in the repsitory

I tried to fix this issues but after fixing on error the next arrisese.

Could you please check the repositor examples

Many thanks

**Comment by chrisfairc** [comment]
I've got the same issue, it would be great if this repo ran out of the box. Will try to fix it but some pointers or a fix from @achlubek would be great! The videos look amazing and the parallax mapping using geometry shader is something I really want to mess around with in opentk - looks like it's been nailed in this project.

**Comment by chrisfairc** [comment]
I ran a git reset --hard 05aa3bda3b86553408cd5790a1dcd978d5f36532 
back a couple of commits and the crashes @bettina4you were seeing stopped - but it looks like it is not finding scene objects now in the samples - do I need to move asset files or export files or something so the engine can load them? Trying some stuff still.

**Comment by achlubek** [comment]
Hi, i will update testing projects with simplier scene and put the media on
the repo to easy this out

17 sty 2018 00:59 "chrisfairc" <notifications@github.com> napisał(a):

**Comment by chrisfairc** [comment]
Cool! I got it so it doesn't crash on the OldCityScene which looks like it has all of its assets there - however I don't see anything when it's running, and this error repeated in the console:

DebugSourceApi DebugTypeError 1282 DebugSeverityHigh 96 Error has been generated. GL error GL_INVALID_OPERATION in (null): (ID: 173538523) Generic error 0

**Comment by achlubek** [comment]
I commited some fixes that made it barely run on GTX 1070, it really barely works and only displays albedo texture, I updated OpenTK version to be 2.0.0 from nuget and fixed some issues with keyboard, also added the model that is now loaded by test scene.

Really, this project is giant spaghetti and it was intended for my to learn opengl, I abandoned it and moved to c++ to start up with better code base, this project should be long dead, I only keep it here because it has meshconverter which I didnt separate yet and I dont have private repos. I wont be supporting this project anymore because this shit is completely broken...

**Comment by chrisfairc** [comment]
Fair enough - you have some nice things in there though for us weirdos who still like opentk =)  Thnx
